src/frontend/pages/upload/[[...params]].tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
```

Comment:
is this file supposed to be named this? lol it looks funny

Reply:
Ahaha yeah Next.js came up with this interesting idea lol

https://nextjs.org/docs/routing/dynamic-routes

Reply:
oh very interesting